src/utils/theme.ts

```diff
@@ -0,0 +1,44 @@
+import { createTheme } from '@shopify/restyle'
+
+const palette = {
+  purpleLight: '#8C6FF7',
+  purplePrimary: '#5A31F4',
+  purpleDark: '#3F22AB',
+
+  greenLight: '#56DCBA',
+  greenPrimary: '#0ECD9D',
+  greenDark: '#0A906E',
+
+  black: '#0B0B0B',
+  white: '#F0F2F3',
+}
+
+const theme = createTheme({
+  colors: {
+    ...palette,
+    mainBackground: palette.white,
+    cardPrimaryBackground: palette.purplePrimary,
+  },
+  spacing: {
+    s: 8,
+    m: 16,
+    l: 24,
+    xl: 40,
+  },
```

Comment:
I have a feeling there aren't enough values and it will limit our flexibility a little bit..

What do you think about adding more values?
```
none: 0,
xxs: 4,
xs: 8,
s: 12,
m: 16,
l: 20,
xl: 24,
xxl: 32,
xxxl: 40
```

Or potentially, keys that are multiples of 4:

```
0: 0,
1: 4,
2: 8,
3: 12,
4: 16,
5: 20,
6: 24,
8: 32,
10: 40
```

(These are what tailwind uses.)

Reply:
I like the numbers, I'll add those

Reply:
Updated